gatsby-ssr.tsx

```diff
@@ -13,7 +13,7 @@ export const onRenderBody = ({ setHeadComponents, setPostBodyComponents }) => {
     },
   };
   const isEnabled =
-    process.env.NODE_ENV === "production" || posthogOptions.isEnabledDevMode;
+    process.env.CONTEXT === "deploy-preview" || posthogOptions.isEnabledDevMode;
```

Comment:
```suggestion
    process.env.CONTEXT === "production" || posthogOptions.isEnabledDevMode;
```

Si le premier test est concluant